docs/api/ref/schemas/product_update_api_v3.yaml

```diff
@@ -11,3 +11,18 @@ properties:
     $ref: ./packagings/packagings-write.yaml
   packagings_complete:
     $ref: ./packagings/packagings_complete.yaml
+  lang:
+    type: string
+    minLength: 2
+    maxLength: 2
+    example: fr
+    description: 2 letter language code of the main language of the product (the most prominent on the packaging)
+  quantity:
+    type: string
+  serving_size:
+    type: string
+  categories_tags:
+    type: array
+    description: An array of categories
+    items:
+      $ref: ./packagings/input_taxonomy_tag.yaml
```

Comment:
So we are making those new docs in parallel to the v2 fields ?

Reply:
I'm not too sure about it. It's likely that v3 fields will evolve and be a bit different, and we don't support the full product fields from the v2 READ API, so that's why I added them here.